packages/flutter/test/cupertino/button_test.dart

```diff
@@ -1004,6 +1009,33 @@ void main() {
       SystemMouseCursors.forbidden,
     );
     await gesture.removePointer();
+
+    // Test focused state mouse cursor
+    await tester.pumpWidget(buildButton(enabled: true, cursor: const _ButtonMouseCursor()));
+    await tester.pump();
+    focusNode.requestFocus();
+    if (focusNode.hasFocus) {
+      await tester.pump();
+      expect(
+        RendererBinding.instance.mouseTracker.debugDeviceActiveCursor(1),
+        SystemMouseCursors.copy,
+      );
+    }
+    focusNode.unfocus();
+    await tester.pump();
+    await gesture.removePointer();
```

Comment:
Minor issue: We don't need to remove pointer after each sub-test. In fact, since we're simulating a mouse, we should probably not remove pointers until the end.

Reply:
Got it will remove it and keep it at the end

Reply:
Addressed